docs/proposals/milestone-1-defaults.md

```diff
@@ -0,0 +1,32 @@
+# Milestone-1 defaults
+
+### Terminology
+Proposed terminology, one of these two terms need to replace `--hub-router`, either `--discovery-node`, 
+`--relay-node` (or something else):
```

Comment:
Before merging, it would be ideal to turn this into a more definitive statement (decision).